api/src/main/java/org/openmrs/module/eptsreports/reporting/library/indicators/HivIndicators.java

```diff
@@ -47,24 +51,24 @@ public CohortIndicator patientEnrolledInHIVStartedARTIndicator(CohortDefinition
 	}
 	
 	public CohortIndicator patientBelow1YearEnrolledInHIVStartedARTIndicatorBeforeOrOnEndDate(CohortDefinition cohortDefinition) {
```

Comment:
I know this was here before, but these indicators actually look the same - they are more of a generic indicator which takes in a cohort and returns an indicator. Please merge into one.

Also pls remove the duplicated parameters of `location` & `locations` - there should just be one for consistency